### PR TITLE
added the ability to invite people to public events

### PR DESCRIPTION
**Pull request by GregJoubert**

### CODE REVIEW

**Comment by minnerthecat** [comment]
Looks good!

File: partEZ/app/Http/Controllers/EventController.php [hunk 1]
```diff
@@ -379,10 +379,16 @@ public function splitEmails($eid)
     {
         $input = Request::all();
         $emails = $input['email-list'];
-
         $emails = array_map('trim', explode(',', $emails));
         self::inviteUsers($emails, $eid);
-        return view('events/success_event');
```
Comment (JenCabral) [inline partEZ/app/Http/Controllers/EventController.php:403]: Why was this removed? Isn't the regular splitEmails still called for non-public emails?

**Comment by JenCabral** [comment]
Other than my comment above, looks good.
